src/series/area.js

```diff
@@ -1,30 +1,21 @@
 import d3 from 'd3';
-import _dataJoin from '../util/dataJoin';
+import dataJoinUtil from '../util/dataJoin';
 import {noop} from '../util/fn';
+import xyBase from './xyBase';
 
 export default function() {
 
-    var decorate = noop,
-        xScale = d3.time.scale(),
-        yScale = d3.scale.linear(),
-        y0Value = d3.functor(0),
-        y1Value = function(d, i) { return d.close; },
-        xValue = function(d, i) { return d.date; };
+    var decorate = noop;
```

Comment:
It didn't feel like it belonged. I thought about putting it on its own component but that seemed overkill.